src/snippets/react-jss.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react';
+import injectSheet from 'react-jss';
+
+const styles = {
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: '100%',
+    width: '100%',
+    backgroundColor: '#f6f9fc'
+  },
+  header: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '40%',
+    minHeight: 250,
+    width: '100%',
+    position: 'relative',
+    justifyContent: 'center'
+  },
+  logoContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: 60,
+    width: 60,
+    borderRadius: 60,
+    backgroundColor: 'white',
+    boxShadow: '0 1px 6px rgba(0, 0, 0, 0.1)',
+    position: 'absolute',
+    top: '1rem',
+    left: '1rem',
+    zIndex: '2'
+  },
+  stripes: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+    width: '100%',
+    height: '100%',
+    overflow: 'hidden',
+    transform: 'skewY(-8deg)',
+    transformOrigin: 0,
+    background: 'linear-gradient(-150deg, #25b57f 0%, #acf0b5 70%)'
+  },
+  stripe: {
+    height: '10vh',
+    overflow: 'hidden',
+    transform: 'skewY(-8deg)',
+    transformOrigin: 0,
+    background:
+    'linear-gradient(-150deg, rgba(255, 255, 255, 0) 40%, #ddecf7 70%)'
+  },
+  stripeBottom: {
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    right: 0,
+    height: '35%',
+    overflow: 'hidden',
+    background: 'linear-gradient(150deg, rgba(172, 240, 181, 0) 40%, #acf0b5 70%)'
+  },
+  titleContainer: {
+    padding: '1rem'
+  },
+  title: {
+    color: 'white',
+    margin: 0,
+    padding: 0,
+    position: 'relative',
+    fontSize: '1.5rem',
+    fontWeight: '100'
+  },
+  subTitle: {
+    extend: 'title',
+    color: '#025450',
+    marginTop: '0.5rem'
+  },
+  form: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+    padding: '2rem',
+    boxSizing: 'border-box'
+  },
+  input: {
+    display: 'block',
+    width: '100%',
+    marginBottom: '1rem',
+    padding: '1.25rem 1rem',
```

Comment:
also here, for the sake of demostration you can write: padding: ['1.25rem', '1rem']